File: backend/admin-api/adminAPI/Models/Country.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace adminAPI.Models
{
    public class Country
    {
        public int CountryID { get; set; }
        public string? CountryName { get; set; }  
        public int CreatedBy { get; set; }
        public DateTime CreatedAt { get; set; }
        public int UpdatedBy { get; set; }
        public DateTime UpdatedAt { get; set; } 
    }
}

File: frontend/customer-app/perkup_admin_app/lib/models/country/country.dart
class Country {
  Country({
    required this.countryID,
    required this.countryName,
    required this.createdBy,
    required this.createdAt,
    required this.updatedBy,
    required this.updatedAt,
  });

  late final int countryID;
  late final String countryName;
  late final int createdBy;
  late final String createdAt;
  late final int updatedBy;
  late final String updatedAt;

  Country.fromJson(Map<String, dynamic> json) {
    countryID = json['countryID'];
    countryName = json['countryName'];
    createdBy = json['createdBy'];
    createdAt = json['createdAt'];
    updatedBy = json['updatedBy'];
    updatedAt = json['updatedAt'];
  }

  Map<String, dynamic> toJson() {
    final data = <String, dynamic>{};
    data['countryID'] = countryID;
    data['countryName'] = countryName;
    data['createdBy'] = createdBy;
    data['createdAt'] = createdAt;
    data['updatedBy'] = updatedBy;
    data['updatedAt'] = updatedAt;
    return data;
  }
}
